package.json

```diff
@@ -20,21 +25,31 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/weilu/bip39.git"
+    "url": "https://github.com/bitcoinjs/bip39.git"
   },
   "license": "ISC",
+  "files": [
+    "src",
+    "types"
+  ],
   "dependencies": {
     "create-hash": "^1.1.0",
     "pbkdf2": "^3.0.9",
     "randombytes": "^2.0.1",
-    "safe-buffer": "^5.0.1",
     "unorm": "^1.3.3"
   },
   "devDependencies": {
+    "@types/create-hash": "1.2.0",
+    "@types/node": "11.11.6",
+    "@types/pbkdf2": "3.0.0",
+    "@types/randombytes": "2.0.0",
+    "@types/unorm": "1.3.27",
```

Comment:
I guess since bip39 doesn't export / return any data types from these, having them in devDependencies should be fine.

Maybe add `@types/node` to dependencies instead, in case someone wants to use Buffer methods. (though imo in that case they should add it to their project...

This is probably fine as is.

Reply:
https://github.com/bitcoinjs/bip39/pull/104/commits/427eb0c0f50914336d9c054a952631cc609f6a08